evergreen.yml

```diff
@@ -301,6 +303,28 @@ functions:
         permissions: public-read
         content_type: application/octet-stream
 
+  "update download center feed":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-powerbi-connector
+        script: |
+          ${prepare_shell}
+          sed -i 's@{RELEASE_VERSION}@'${RELEASE_VERSION}'@' mongo-powerbi-connector-downloads_template.json
+          sed -i 's@{RELEASE_PQX_PATH}@'${RELEASE_PQX_PATH}'@' mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+          cat mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+    - command: s3.put
+      params:
+        aws_key: ${release_aws_key}
+        aws_secret: ${release_aws_secret}
+        local_file: mongo-tableau-connector/mongo-tableau-connector-downloads_template.json
+        remote_file: mongo-tableau-connector/mongo-tableau-connector-downloads.json
+        bucket: translators-connectors-releases
+        permissions: public-read
+        content_type: application/json
```

Comment:
Should this be `application/json`? If it is signifying the download type for the pqx file, it should be [`application/octet-stream`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)

Reply:
No, it means that the file `mongo-powerbi-connector-downloads.json` is a json file